internal/backends/winit/renderer/sw.rs

```diff
@@ -7,17 +7,24 @@ use super::WinitCompatibleCanvas;
 use i_slint_core::api::PhysicalSize as PhysicalWindowSize;
 use i_slint_core::graphics::Rgb8Pixel;
 use i_slint_core::lengths::LogicalLength;
-pub use i_slint_core::software_renderer::SoftwareRenderer;
+use i_slint_core::renderer::Renderer;
+pub use i_slint_core::software_renderer::SoftwareRenderer as CoreSoftwareRenderer;
 use i_slint_core::window::WindowAdapter;
 use std::cell::RefCell;
 use std::rc::{Rc, Weak};
 
-impl<const BUFFER_COUNT: usize> super::WinitCompatibleRenderer for SoftwareRenderer<BUFFER_COUNT> {
+pub struct SoftwareRenderer<const MAX_BUFFER_AGE: usize> {
```

Comment:
Why not call this WinitSoftwareRendererorsomething, then you don't need to rename the Core one.

Reply:
Good idea, will do